deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/ibm-block-csi-operator.v1.6.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,711 @@
+      mediatype: image/png
+  labels:
+    alm-owner-ibmblockcsi: ibmblockcsioperator
+    operated-by: ibmblockcsioperator
+  selector:
+    matchLabels:
+      alm-owner-ibmblockcsi: ibmblockcsioperator
+      operated-by: ibmblockcsioperator
+  installModes:
+    - type: OwnNamespace
+      supported: true
+    - type: SingleNamespace
+      supported: true
+    - type: MultiNamespace
+      supported: false
+    - type: AllNamespaces
+      supported: false
+  customresourcedefinitions:
+    owned:
+      - name: ibmblockcsis.csi.ibm.com
+        version: v1
+        group: csi.ibm.com
+        kind: IBMBlockCSI
+        displayName: "IBM block storage CSI driver"
+        description: "Represents an block storage CSI driver"
```

Comment:
we should remember to change `an block` to `a block` (same for certified CSV yaml) - should I open a ticket for it?